docs/package.json

```diff
@@ -1,16 +1,12 @@
 {
   "name": "docs",
-  "version": "0.1.0",
+  "private": true,
   "devDependencies": {
-    "gitbook-cli": "2.3.0",
-    "markdownlint-cli": "0.3.1"
+    "markdownlint-cli": "^0.18.0",
+    "shx": "^0.3.2"
   },
   "scripts": {
-    "build": "yarn update-version && gitbook build",
-    "lint": "bash ../build/markdown-linter.sh",
-    "start": "gitbook serve",
-    "dist": "yarn update-version && yarn docker-gitbook-build",
-    "update-version": "VERSION=`git describe --tags --always` && sed -i -- 's/_pre-release_/'$VERSION'/g' ./version.md",
-    "docker-gitbook-build": "docker run -v $(pwd):/usr/src/gitbook nerc/gitbook-builder gitbook build"
+    "lint": "markdownlint src/**/*.md",
```

Comment:
Does this lint target pick up the linting errors that codeclimate has raised? If not, why not?

Reply:
It doesn't I will check.